### ISSUE TITLE
if isCenterloss:     lambda_c = 0.2  # this parama is variable     input_target = Input(shape=(1,))  # single value ground truth labels as inputs     centers = Embedding(OUT_DIMS, Feats_Dims)(input_target)     l2_loss = Lambda(lambda x: K.sum(K.square(x[0] - x[1][:, 0]), 1, keepdims=True), name='l2_loss')([fc, centers])     model_centerloss = Model(inputs=[resnet_base.input, input_target], outputs=[x, l2_loss])     model_centerloss.compile(optimizer=SGD(lr=0.05), loss=["categorical_crossentropy", lambda y_true, y_pred: y_pred],                              loss_weights=[1, lambda_c], metrics=['accuracy'])

### ISSUE THREAD

**Issue by FlyEgle** [keras-team/keras#9880]
Please make sure that the boxes below are checked before you submit your issue. If your issue is an implementation question, please ask your question on [StackOverflow](http://stackoverflow.com/questions/tagged/keras) or [join the Keras Slack channel](https://keras-slack-autojoin.herokuapp.com/) and ask there instead of filing a GitHub issue.

Thank you!

- [ ] Check that you are up-to-date with the master branch of Keras. You can update with:
pip install git+git://github.com/keras-team/keras.git --upgrade --no-deps

- [ ] If running on TensorFlow, check that you are up-to-date with the latest version. The installation instructions can be found [here](https://www.tensorflow.org/get_started/os_setup).

- [ ] If running on Theano, check that you are up-to-date with the master branch of Theano. You can update with:
pip install git+git://github.com/Theano/Theano.git --upgrade --no-deps

- [ ] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).

**Comment by FlyEgle** [comment]
if isCenterloss:
    lambda_c = 0.2  # this parama is variable
    input_target = Input(shape=(1,))  # single value ground truth labels as inputs
    centers = Embedding(OUT_DIMS, Feats_Dims)(input_target)
    l2_loss = Lambda(lambda x: K.sum(K.square(x[0] - x[1][:, 0]), 1, keepdims=True), name='l2_loss')([fc, centers])
    model_centerloss = Model(inputs=[resnet_base.input, input_target], outputs=[x, l2_loss])
    model_centerloss.compile(optimizer=SGD(lr=0.05), loss=["categorical_crossentropy", lambda y_true, y_pred: y_pred],
                             loss_weights=[1, lambda_c], metrics=['accuracy']
